### PR TITLE
Feat: change the pagination to cursor and add sorter --- Multi collections 

### PR DESCRIPTION
**Pull request by JeffreyMa597**
What problem does this PR solve?
------
This PR is for the page of `Multi-collections`.

Ref: https://github.com/Magickbase/godwoken_explorer/issues/556

Check List
------
1、sorted by total supply/circulating supply
2、sorted by holders
3、sorted by name
4、filtered by name (fuzzy)

### Test
e2e Test
### Task
none

### CODE REVIEW

**Comment by Keith-CY** [comment]
image

**Comment by JeffreyMa597** [comment]
@Naupio Please also pay attention to my comment under this issue. we missing a sorter in the API of this page. Thanks a lot.

**Comment by JeffreyMa597** [comment]
@Naupio Please post your PRs, if you finish the update. So, I can track it. Thanks a lot.

**Review by qiweiii — APPROVED** [review]

**Comment by FrederLu** [comment]
`Multi-token` is displayed in ascending order by default on the page, but the interface is displayed in descending order by default. When the front end requests, a default sorting can be given according to the page display situation.
image
image
image

File: utils/handler.ts [hunk 1]
```diff
@@ -73,3 +73,25 @@ export const handleCopy = async (value: string) => {
 export const handleNftImageLoadError = (e: React.SyntheticEvent<HTMLImageElement, Event>) => {
   e.currentTarget.src = '/images/nft-placeholder.svg'
 }
+
+export const handleSorterArrayFromUrl = (url, sorters) => {
+  const paramArr = url.slice(url.indexOf('?') + 1).split('&')
```
Comment (Keith-CY) [inline utils/handler.ts:78]: Use `URLSearchParams` to parse query

Ref: https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams

File: pages/multi-token-collections/index.tsx [hunk 2]
```diff
@@ -75,25 +96,71 @@ const fetchErc1155List = (variables: Variables): Promise<MultiTokenCollectionLis
...
+    const sorterParams = []
+
+    paramArr.map(param => {
+      const [key, val] = param.split('=')
+      const decodeVal = decodeURIComponent(val)
+
+      sorters.includes(key) && sorterParams.push({ sort_type: decodeVal, sort_value: UdtsSorterValueEnum[key] })
+    })
+    return sorterParams
+  }
```
Comment (Keith-CY) [inline pages/multi-token-collections/index.tsx:130]: A similar method was defined in https://github.com/Magickbase/godwoken-explorer-ui/pull/738/files#diff-dbc569892fd8b550b4a391217df328947c838a65ff30d015e970a47670e5280fR77

**Comment by Keith-CY** [comment]
I see some conversation are not resolved, how is it going @JeffreyMa597

**Comment by Keith-CY** [comment]
Any update?

**Review by FrederLu — APPROVED** [review]

**Review by Keith-CY — APPROVED** [review]

**Pull request merged**